index.js

```diff
@@ -41,3 +41,61 @@ app.get('/dogs', (req, res) => {
         })
     })
 })
+
+app.put('/dogs', jsonParser, (req, res) => {
+    if (req.body.winnerID == '') {
+        return res.status(400).json({
+            success: false,
+            message: 'Please ensure request has valid "winnerID" and "winCount" and try again.',
+            data: []
+        })
+    }
+    mongoClient.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true }, (err, client) => {
+        if (err) {
+            return res.status(500).json({
+                success: false,
+                message: 'Error connecting to database.',
+                data: []
+            })
+        }
+        let db = client.db(dbName)
+        try {
+            declareChampion(db, req.body, function (result) {
+                res.status(200).json({
+                    success: true,
+                    message: 'Your choice has been received and the database has been updated.',
+                    data: result
+                })
+            })
+        }
+        catch (err) {
+            return res.status(400).json({
+                success: false,
+                message: 'Not able to update database',
+                data: []
+            })
+        }
+    })
+})
+
+const declareChampion = async function (db, details, callback) {
+    let collection = db.collection(collectionName)
+    try {
+        let winner = await collection.find({"_id": ObjectId(details.winnerID)}).toArray()
+        if (!winner[0].hasOwnProperty('winCount')) {
+            winner[0].winCount = 0
+        }
+        let newCount = parseInt(winner[0].winCount) + 1
+        collection.updateOne(
+            {"_id": ObjectId(winner[0]._id)},
+            {$set: {"winCount": newCount}},
```

Comment:
You could try using `$inc` https://docs.mongodb.com/manual/reference/operator/update/inc/#up._S_inc